src/main/webapp/data-visualization.js

```diff
@@ -133,21 +132,28 @@ function createDataArray(censusDataArray, isCountyQuery) {
   const vizDataArray = [];
   // first row is headers
   const regionIndex = censusDataArray[0].indexOf('state');
-  censusDataArray = censusDataArray.splice(1); // get rid of header row
+  censusDataArray = censusDataArray.slice(1); // get rid of header row
   // Check to see if an extra calculation for percentages is needed
+  const table = initDataTable(isCountyQuery);
   if (checkPercentage(censusDataArray[0])) {
```

Comment:
I updated this a bit (checkPercentage now also takes isCountyQuery as a parameter) but I guess that'll get merged when we merge data-table